### ISSUE TITLE
Scheduled shutdown Lambda not working - Unable to import module 'shutdown': No module named 'shutdown'

### ISSUE THREAD

**Issue by shantgup** [aws-samples/amazon-sagemaker-canvas-service-catalog#1]
Hello,

Creating this issue on behalf of an Enterprise customer. 

Customer followed the [blog](https://aws.amazon.com/blogs/machine-learning/provision-and-manage-ml-environments-with-amazon-sagemaker-canvas-using-aws-cdk-and-aws-service-catalog/) step-by-step to deploy the solution.

After deploying and provisioning the product for the scheduled shutdown, the Lambda function does not run. And in the CloudWatch Logs for the Lambda function we see the below error. 

```
2023-03-13T12:10:22.478-05:00
[ERROR] Runtime.ImportModuleError: Unable to import module 'shutdown': No module named 'shutdown'
Traceback (most recent call last):
```

Customer went through the blog a second time as well, but still getting the same error. 

Please advise on how to fix, and if this is something that has broken since the blog was released in September 2022, please update it. 

I have shared this GitHub issue with the customer, please let me know if I can provide any other information.

**Comment by dgallitelli** [comment]
I tried to deploy the solution via `sh deploy_stack.sh` and it worked like a charm, haven't been able to reproduce the issue mentioned above. Can you provide CDK version? Tried with CDK v2.66.1 and NPM v9.5.1, as well as CDK v2.69.0 and NPM v9.6.2.

Are you executing the command from the root folder?

**Comment by dgallitelli** [comment]
nevermind, found the problem, solved with commit [1193950](https://github.com/aws-samples/amazon-sagemaker-canvas-service-catalog/commit/1193950df48fff2d0116fad5a42f985c5d87a476). Please validate with customer. Feel free to re-open if that's not the case.
